_topic_map.yml

```diff
@@ -1531,3 +1531,17 @@ Topics:
     File: v1.User
   - Name: v1.UserIdentityMapping
     File: v1.UserIdentityMapping
+---
+Name: Service Mesh Install
```

Comment:
@kalexand-rh, the content plan we'd agreed on is to deliver our docs separately until GA because for Tech Preview we're a separate install and on a separate release cycle.